ui/app/pages/settings/security-tab/security-tab.component.js

```diff
@@ -17,6 +17,8 @@ export default class SecurityTab extends PureComponent {
     setParticipateInMetaMetrics: PropTypes.func,
     showIncomingTransactions: PropTypes.bool,
     setShowIncomingTransactionsFeatureFlag: PropTypes.func,
+    setUsePhishDetect: PropTypes.func,
```

Comment:
These two props should both be required - it looks like they should be defined in all cases.

Reply:
Definitely can do, just curious though, would the other existing props (especially related to metametrics) need to be required in this case?

Reply:
We haven't been diligent with this in most of our UI, but it's something we're trying to do going forward. We might eventually enable the `react/require-default-props` rule.

We've been slow to update existing components to follow this convention, but feel free to add `isRequired` on any obvious ones you find. The functions from `mapDispatchToProps` like the metametrics function are a great example - those should always be required.

Reply:
Cool, got it. I've added `isRequired` to those two and others mentioned